### ISSUE TITLE
[MultiComboBox]: Accessibility: Checkboxes within the MultiComboBox should be of at least 24px as per guidelines

### ISSUE THREAD

**Issue by abhijeet181088** [SAP/ui5-webcomponents#9019]
### Describe the bug

Currently the size of checkboxes within MultiComboBox are following:
Normal: 22px (1.375rem)
Compact: 16px (1rem)

Accessibility guidelines for size of target elements:
**_"Targets have an area of at least a 24 CSS pixels diameter circle, including dedicated surrounding space, unless it is within a sentence or block of text."_**

> 

### Isolated Example

https://stackblitz.com/edit/github-pngbdn?file=src%2FApp.tsx

### Reproduction steps

1. Go to the isolated stackblitz example link provided.
2. Click on the first MultiComboBox (normal) and inspect the checkbox within it.
3. Size of checkbox is 22px (1.375rem)
4. Click on the second MultiComboBox (compact) and inspect the checkbox within it.
5. Size of checkbox is 16px (1rem)
...

### Expected Behaviour

Size of checkboxes should be at least 24px as per accessibility guidelines.

### UI5 Web Components for React Version

1.28.1

### UI5 Web Components Version

1.24.0

### Browser

Chrome

### Operating System

macOS

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-rl,
Could you please look into this issue?

Regards,
Todor

**Comment by niyap** [comment]
Hello @abhijeet181088,

Thanks for sharing your findings!
Actually, "Targets have an area of at least a 24 CSS pixels diameter circle, including dedicated surrounding space, unless it is within a sentence or block of text." means the checkbox together with its touch area.
So, the actual size currently is 44x40(cozy) 
<img width="575" alt="image" src="https://github.com/SAP/ui5-webcomponents/assets/38278268/6e7f0c92-73a2-4102-927c-ddf2f22d21d6">
and 32x32 (compact) 
<img width="619" alt="image" src="https://github.com/SAP/ui5-webcomponents/assets/38278268/8ea7e314-4073-4724-934a-07018a09654a">
 so the requirement is fulfilled.

Kind Regards,
Niya
